shell/platform/darwin/macos/framework/Headers/FLEEngine.h

```diff
@@ -26,34 +26,46 @@ FLUTTER_EXPORT
 /**
  * Initializes an engine with the given viewController.
  *
- * @param viewController The view controller associated with this engine. If nil, the engine
- *                       will be run headless.
+ * @param labelPrefix Currently unused; in the future, may be used for labelling threads
```

Comment:
would it help to name this "theadLabelPrefix"?

Reply:
I'm deliberately using the same name as iOS, since I am hoping to merge the public engine header between macOS and iOS sometime soon.